src/compiler/crystal/compiler.cr

```diff
@@ -98,13 +98,21 @@ module Crystal
     # and can later be used to generate API docs.
     property? wants_doc = false
 
-    # Can be set to an array of strings to emit other files other
+    @[Flags]
+    enum EmitKinds
```

Comment:
Better have the name in singular, that's used throughout the code base.
Maybe `EmitTarget` would even be a better name.

Reply:
And I'd suggest to have the values either all uppercase or all lower case. I prefer the former.